content/docs/manage/projects.md

```diff
@@ -112,6 +123,47 @@ The costs associated with a shared project are charged to the Neon account that
 
 For additional information, refer to our [Project sharing guide](/docs/guides/project-sharing-guide).
 
+## Advanced settings
+
+From the **Settings** page, you can also set defaults or apply bulk changes across your Neon objects:
+- [Reset default compute size](#reset-the-default-compute-size) (Pro plan only)
+- [Configure history retention range](#configure-history-retention)
+
+### Reset the default compute size
+
+_Compute size_ is the number of Compute Units (CUs) assigned to a Neon compute endpoint. The number of CUs determines the processing capacity of the compute endpoint. One CU is equal to 1 vCPU with 4 GBs of RAM. Currently, a Neon compute endpoint can have anywhere from .25 CUs to 7 CUs. Larger compute sizes will be supported in a future release.
+
+For [Free Tier](/docs/introduction/free-tier) users, Neon sets your compute size to a static shared 1 vCPU, with 1 GB of RAM for each branch (max 10 branches).
+
+[Pro plan](/docs/introduction/pro-plan) users have the flexibility to choose a compute size for each branch's compute endpoint individually. By default, new branches inherit the compute endpoint size from your first branch (i.e., `main`). However, there may be times when you want to reset this default. For example, if you want to create a series of read-only replicas, where each replica typically requires less compute per branch.
+
+To reset the default compute size, go to **Settings** > **Compute**.
+
+You can choose from two options:
+
+- **Fixed Size:** Select a fixed compute size ranging from .25 CUs to 7 CUs. A fixed-size compute does not scale to meet workload demand.
+- **Autoscaling:** This option allows you to specify a minimum and maximum compute size. Neon scales the compute size up and down within the selected compute size boundaries in response to the current load. Currently, _Autoscaling_ supports a range of 1/4 (.25) CU to 7 CUs. The 1/4 CU and 1/2 CU settings are _shared compute_. For information about how Neon implements the _Autoscaling_ feature, see [Autoscaling](/docs/introduction/autoscaling).
+
+_Example: default minimum and maximum autoscale settings_
+
+![Default autoscaling min and max](/docs/manage/default_autoscale.png)
```

Comment:
Not sure why, but the items selected in the sidebar in the image are hard to see.